docs/_advanced/lazy_listener.md

```diff
@@ -0,0 +1,79 @@
+---
+title: Lazy listeners (FaaS)
+lang: en
+slug: lazy-listeners
+order: 9
+---
+
+<div class="section-content">
+⚠️ Lazy listener functions are a beta feature to make it easier to deploy Bolt for Python apps to FaaS environments. As the feature is developed, Bolt for Python's API is subject to change.
+
+Typically you'd call `ack()` as the first step of your listener functions. Calling `ack()` tells Slack that you've received the event and are handling it in within reasonable amount of time (3 seconds).
+
+However, apps running on FaaS or similar runtimes that don't allow you to run threads or processes after returning an HTTP response cannot follow this pattern. Instead, you should set the `process_before_response` flag to `true`. This allows you to call ack() and handle the event in seperate threads or processes. For events that do not require acknowledgement, you can create the listener as you normally would.
```

Comment:
Please improve the wordings but the latter half is not correct. As mentioned in the original version, `process_before_response` doesn't let you free from 3-second timeouts in both Bolt JS and Python. It just holds off sending HTTP responses until the function completes.
```suggestion
However, apps running on FaaS or similar runtimes that don't allow you to run threads or processes after returning an HTTP response cannot follow this pattern. Instead, you should set the `process_before_response` flag to `True`. While this allows you to run your app on FaaS, the app still has the limitation to complete everything within 3 seconds.
```